sass/components/_header.scss

```diff
@@ -124,3 +124,145 @@
   @include govuk-width-container;
   border: none;
 }
+
+.dps-header {
```

Comment:
We renamed classes to `.fallback-dps-header` with https://github.com/ministryofjustice/manage-key-workers/pull/771/files#diff-bce0b0d4d2d98fa5ccab48b82880ead0c431ee429709ae27cd7c079326a96ff2R1 so I'm adding the previously named classes so React pages still work